common/src/main/scala/org/apache/celeborn/common/RssConf.scala

```diff
@@ -370,6 +371,83 @@ class RssConf(loadDefaults: Boolean) extends Cloneable with Logging with Seriali
   def workerRecoverPath: String = get(WORKER_RECOVER_PATH)
   def partitionSorterCloseAwaitTimeMs: Long = get(PARTITION_SORTER_SHUTDOWN_TIMEOUT)
   def workerDiskFlusherShutdownTimeoutMs: Long = get(WORKER_FLUSHER_SHUTDOWN_TIMEOUT)
+  def commitFilesTimeout: Long = get(WORKER_SHUFFLE_COMMIT_TIMEOUT)
+  def writerTimeoutMs: Long = get(WORKER_WRITER_CLOSE_TIMEOUT)
+  def HDDFlusherThreads: Int = get(WORKER_FLUSHER_HDD_THREADS)
+  def SSDFlusherThreads: Int = get(WORKER_FLUSHER_SSD_THREADS)
+  def hdfsFlusherThreads: Int = get(WORKER_FLUSHER_HDFS_THREADS)
+  def flushAvgTimeWindow: Int = get(WORKER_FLUSHER_AVGFLUSHTIME_SLIDINGWINDOW_SIZE)
+  def flushAvgTimeMinimumCount: Int = get(WORKER_FLUSHER_AVGFLUSHTIME_SLIDINGWINDOW_MINCOUNT)
+  def diskMinimumReserveSize: Long = get(WORKER_DISK_RESERVE_SIZE)
```

Comment:
diskReserveSize, method name should be consistent with config name, plz review other places.